### PR TITLE
Blogs

### PR DESCRIPTION
**Pull request by jhinkey**
Hi Rich,

Here's the revamped Blogs section for 7.0.

https://issues.liferay.com/browse/LRDOCS-1906

Thanks,
Jim

### CODE REVIEW

File: discover/portal/articles/07-collaborating/01-publishing-blogs.markdown [hunk 1]
```diff
@@ -0,0 +1,444 @@
...
+you're more likely to draw readers to your blog entry by decorating it with an
+enticing cover image! 
+
+By now you're probably chomping at the bit to start blogging, right?! Let's get
+started. 
+
+## Adding Blog Entries [](id=adding-blog-entries)
+
+Each site comes with a built-in blog instance. So let's explore adding an entry
+to your site's blog. The Blogs Administration portlet provides the best place to
```
Comment (sergiogonzalez) [inline discover/portal/articles/07-collaborating/01-publishing-blogs.markdown]: I think that we don't have the concept of Blogs Administration portlet. I mean, we do, but from a developer point of view. From a user point of view is just blogs in Site Administration. But maybe people will understand what Blogs Administration mean.

File: discover/portal/articles/07-collaborating/01-publishing-blogs.markdown [hunk 2]
```diff
@@ -0,0 +1,444 @@
...
+yourself, "Where are the editor's controls?" The editor gives you a seamless
+writing experience, displaying controls when you need them, and hiding them from
+view when you don't need them. This keeps your canvas unclutterd, so you can
+focus on writing. As you create content, context-specific controls appear. 
+
+Go ahead and enter text in the *Content* area. If you highlight text, controls
+appear. These controls let you style the text or convert it to a link or a
+tweet--this shares your blog post on Twitter. 
+
+Whenever you park your cursor in the content area, the *Add* icon (`+`)
```
Comment (sergiogonzalez) [inline discover/portal/articles/07-collaborating/01-publishing-blogs.markdown]: Besides clicking the + icon to add images, I think it's also interesting to mention that users can also add images to blog content by dragging a file from desktop and drop it in the blog entry content. At that point, the image will start to be uploaded and will be already part of the blog entry.

File: discover/portal/articles/07-collaborating/01-publishing-blogs.markdown [hunk 3]
```diff
@@ -0,0 +1,444 @@
...
+how to specify your blog entry's other characteristics. 
+
+The edit screen's first input field--*Drag \& Drop to Upload*--lets you add a
+cover image (optional) for your entry. This might be an image that represents
+your entry's purpose and attract readers. You can drag and drop onto this field
+any image you like. As an alternative to dragging and dropping an image, you
+click on the *Select File* button to use the image selector to choose an
+existing image in the blog, an image file from Documents and Media, or an image
+outside the portal. You can browse to a file and upload it, or you can specify
+an image's URL for the image selector to take in the image automatically. After
```
Comment (sergiogonzalez) [inline discover/portal/articles/07-collaborating/01-publishing-blogs.markdown]: Regarding the Cover Image I think that right now we don't support specifying a image via URL. It's something that we have in mind but we didn't have the time to do it yet, so I'd skip that for now.

File: discover/portal/articles/07-collaborating/01-publishing-blogs.markdown [hunk 4]
```diff
@@ -0,0 +1,444 @@
...
+users. For example, you can add the Tags Navigation portlet to another column on
+your blogs page, allowing users to browse blog entries by tag. 
+
+Below this is the Related Assets heading. If there's some other content in the
+portal that's related to your entry, you can choose it here. For example, you
+might want to write a blog entry talking about a particular discussion that
+happened on the forums. To link those two assets together, select the forum
+thread under Related Assets. 
+
+Blog entries also support trackbacks and pingbacks. Trackbacks are
```
Comment (sergiogonzalez) [inline discover/portal/articles/07-collaborating/01-publishing-blogs.markdown]: Regarding trackbacks, we have disabled those by default in portal.properties (they are really all and most of the new blogs don't support it anymore). Maybe we sholdn't include it here, or at least indicate that is deprecated?
We support pingbacks though, so maybe we can give more relevance to Pingback than trackbacks?

**Comment by jhinkey** [comment]
Thanks for your input, @sergiogonzalez . I pushed up a commit with changes that address your review comments .

@sez11a - I also refactored "Liferay Portal" and "portlets" references.

**Comment by sez11a** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by sez11a** [comment]
Superseded by https://github.com/jhinkey/liferay-docs/pull/285

**Pull request closed without merging**